src/modules/Dashboard/features/RespondentData/RespondentDataSummary/Report/Charts/ScatterChart/ChartTooltip/ChartTooltip.utils.test.tsx

```diff
@@ -5,7 +5,12 @@ import { getReviewOption } from './ChartTooltip.utils';
 describe('getReviewOption', () => {
   test.each([
     // [mine, other, expected]
-    [0, 0, null, 'returns null when both mine and other are 0'],
+    [
+      0,
+      0,
+      t('leaveReview'),
+      'returns correct translation for leave review when both mine and other are 0',
```

Comment:
let's move 4th arguments on first place as `description` for test cases and change the row 32 accordingly:
`])('correctly processes mine: %i, other: %i', (mine, other, expected) => {` 
=> 
`])('%d', (_description, mine, other, expected) => {`